src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -166,32 +155,27 @@ EnableBiometry.navigationOptions = nuxNavigationOptions
 
 const styles = StyleSheet.create({
   container: {
-    paddingTop: 186,
+    paddingTop: 72,
     paddingHorizontal: 40,
     alignItems: 'center',
   },
-  containerLeftAligned: {
-    alignItems: 'flex-start',
-  },
   contentContainer: {
     flex: 1,
     backgroundColor: colors.onboardingBackground,
   },
-  imageContainer: {
-    marginBottom: Spacing.Thick24,
-  },
-  description: {
-    ...fontStyles.regular,
-    textAlign: 'center',
-    marginBottom: Spacing.Thick24,
+  biometryButton: {
+    width: '100%',
```

Comment:
I suspect that this is unnecessary b/c of the `size={BtnSizes.FULL}` on line 139

Reply:
looks like this without it
<img width="299" alt="Screenshot 2023-02-21 at 1 10 14 PM" src="https://user-images.githubusercontent.com/7979215/220459219-78f44228-1918-428b-899b-bea61caf9e6f.png">

Reply:
i futzed with this for a while. I think it's because of [this line](https://github.com/valora-inc/wallet/blob/caefb3dbb968e7da5c5466a7e69ad9d5012adfea/src/components/Button.tsx#L199) in the Button component. When I added a View wrapping the button and made its flexDirection "row", it sort of worked, but the width was incorrect. Which led me to using 100% width, at which point it was possible to remove the wrapper with no effect (somehow...)

Reply:
the weird thing is that in other onboarding steps, BtnSizes.FULL is sufficient to get the full screen appearance that we want here. so I don't really get it.

Reply:
weird, but if it works 👍